### DISCUSSION TITLE
Why can't RTCDataChannel transfer byte arrays directly?

### DISCUSSION THREAD

**Discussion by TenSpaces** [devopvoid/webrtc-java#130] [Q&A]
I want to transmit JSON strings continuously through RTCDataChannel, and the size of the string may be different each time, so I use RTCDatachannel. send(RTCDataChannelBuffer buffer), But I found there is a RTCDataChannel method can direct transmission byte, it is RTCDataChannel sendByteArrayBuffer (byte [] buffer, Boolean binary), but this method is private.I wonder if this method should be set as shared?

**Comment by devopvoid** [comment]
You can use `ByteBuffer` to achieve the same. Please refer to the documentation [here](https://jrtc.dev/#/guide/data_channels?id=sending-and-receiving-data).
